.gitignore

```diff
@@ -1,7 +1,7 @@
 # Built application files
 *.apk
 *.ap_
-
+.idea
```

Comment:
The link that YOU posted says that we SHOULD check-in the .idea directory!  Make up your mind!